.jshintrc

```diff
@@ -30,5 +29,5 @@
   "white": false,
   "eqnull": true,
   "esnext": true,
-  "unused": "vars"
+  "unused": true
```

Comment:
I like keeping this "vars", I find it nice to be able to reference expected params even if they're not used. Am I an outsider on this one? O.o